arbeitszeit_flask/views/pay_means_of_production.py

```diff
@@ -23,8 +23,14 @@ class PayMeansOfProductionView:
 
     def respond_to_get(self, form: PayMeansOfProductionForm) -> Response:
         plan_id: Optional[str] = request.args.get("plan_id")
+        amount: Optional[str] = request.args.get("amount")
+        type_of_payment: Optional[str] = request.args.get("type_of_payment")
         if plan_id:
             form.plan_id_field().set_value(plan_id)
+        if amount:
```

Comment:
This looks wrong to me. Shouldn't the form already be constructed with the correct fields in it?